data/intuos-pro-2-s.tablet

```diff
@@ -0,0 +1,51 @@
+# Wacom
+# Intuos Pro 2 S
+# PTH-460
+#
+# Button Map:
+# (A=1, B=2, C=3, ...)
+#
+#    *-----------------------*
+#    |                       |
+#  A |                       |
+#  B |                       |
+#  C |                       |
+#  G |        TABLET         |
+#  D |                       |
+#  E |                       |
+#  F |                       |
+#    |                       |
+#    *-----------------------*
+#
+# Touch Ring Map:
+# (A=1st ring, B=2nd ring, ...)
+#
+#    *-----------------------*
+#    |                       |
+#  A |        TABLET         |
+#    |                       |
+#    *-----------------------*
+#
+# LED Map:
+# No changeable LEDs or modes
+
+[Device]
+Name=Wacom Intuos Pro S
+DeviceMatch=usb:056a:0392;bluetooth:056a:0393;
+Class=Intuos5
+Width=6
+Height=4
+Layout=intuos-pro-2-s.svg
+IntegratedIn=
+Styli=0x842;0x84a;0x160802;0x16080a;0x100802;0x10080a;0x140802;0x14080a;0x120802;0x100804;0x10080c;0x100902;0x10090a;0x80842;0x100842;0x10084a
```

Comment:
23e33ffbe717063bf537cfc38b002e52443759f7 started sorting all the IDs in numerical order, can you do the same here please?